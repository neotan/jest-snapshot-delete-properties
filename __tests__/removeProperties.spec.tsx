import * as React from "react";
import snapshotRemoveProperties from "../";
import { shallow } from "enzyme";

expect.addSnapshotSerializer(snapshotRemoveProperties(["data-remove"]));

test("should remove attributes", () => {
  const component = shallow(
    <div data-remove="remove" id="div1">
      div1
    </div>
  );

  expect(component).toMatchSnapshot();
});
