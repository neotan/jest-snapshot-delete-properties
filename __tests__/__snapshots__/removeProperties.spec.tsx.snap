// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should remove attributes 1`] = `
<div
  id="div1"
>
  div1
</div>
`;
